### PR TITLE
Bookstore: initialize project with components

### PR DESCRIPTION
**Pull request by fabgrel10**
In this PR:

- [x] Initialize React app.
- [x] Add React Redux.
- [x] Structure application files using a "feature folder" approach and the [ducks pattern](https://github.com/erikras/ducks-modular-redux).
- [x] Add [React Router](https://reactrouter.com/web/guides/quick-start) and set two routes: one for Bookstore and one for Categories.

### CODE REVIEW

**Review by Meltrust — COMMENTED** [review]
Hi @fabgrel10,

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## Invalid Code Review Request

You have submitted a project, that uses typescript, instead of plain Js, React, and Redux as stated on the project requirements.

However, 👏 👏 👏  for the effort that you have put into using/learning different technologies.

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

**Review by Davidosky007 — COMMENTED** [review]
Hello @fabgrel10  👋,

_This is an additional review as a result of the second opinion request._

Hi @fabgrel10 it is important the technologies required for this project are used this is important for uniformity sake, exploring other technologies and frameworks is also good but you can always refactor your old projects during your free time thanks to the foundational knowledge you can easily learn more technologies and frameworks. however, typescript might be considered in the future but for now, you need to follow the project requirements.

As this was the only reason that your project was not approved it will be approved as soon as possible.
Please make the necessary changes and ask for another review in your Dashboard and the next Code Reviewer will mark it as approved right away :)

Keep rocking 🚀!

**Comment by fabgrel10** [comment]
@Meltrust @Davidosky007 I understand. I temporarily refactored the project to JS only to accomplish Microverse requirements.

File: src/pages/BooksPage.jsx [hunk 1]
```diff
@@ -0,0 +1,12 @@
+import { Link } from 'react-router-dom';
+
+const BooksPage = () => {
+  return (
+    <div>
+      <h1>Books Page</h1>
+      <Link to="/categories">Categories</Link>
+    </div>
+  );
+};
```
Comment (mikethreels) [inline src/pages/BooksPage.jsx:10]: - [ ] according to the requirements BooksPage - which is the default view should display the list of books (empty at this point but it should be ready for the data) with the Remove button (no funcionality yet) and it Should have a form for adding a book (no functionality yet).
- [ ] please make sure The building blocks of your app (namely the `Book` and `CreateBook` components) should be set as re-usable components.

**Review by mikethreels — CHANGES_REQUESTED** [review]
Hi Fabricio, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Required Changes ♻️
_Check the comments under the review._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/pages/BooksPage.jsx (hunk 1 above)
Comment (fabgrel10) [reply src/pages/BooksPage.jsx]: My bad @mikethreels I forgot to refactor and my BookForm and BookList components. It's all working fine now.

**Review by rloterh — APPROVED** [review]
## Status: Approved :heavy_check_mark:

Hi Fabricio, 

Your project is complete! Great job for making the necessary changes recommended in the previous review. There is nothing else to say other than... it's time to merge it 🍾 :shipit:. Congratulations! 🎉🥂

<br>
Cheers and Happy coding!👏 💻 🍷 

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have 1 more limited review per this project. If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**